### ISSUE TITLE
List of supported XEP's in readme.md

### ISSUE THREAD

**Issue by strixaluco** [TelepathyQt/telepathy-nonsense#6]
It would be really useful to be able to track a list of supported XEP's in readme.md. For instance, like here: https://github.com/esl/MongooseIM#features-and-supported-standards
Is there any similar list with implemented XEP's? I'd submit PR if there was such a list as a reference.

**Comment by Kaffeine** [comment]
Hello!

I'm agree for 100% that we need such list, but we actually need a bit more verbose list, because most of XEP's should be implemented in Telepathy, QXmpp and, finally, in Nonsense.
I tried to add a wiki page and create a table with columns "Supported by Telepathy", "Supported by QXmpp" and "Supported by Nonsense" and coloured cells (green for implemented, red for not implemented features) and vertical span for some features (e.g. for FileTransfer has one Telepathy spec, but a number of XEP's), but was disappointed by very poor table support on github (I tried markdown and MediaWiki formats) and did not save the table.

I hope that we will have a proper wiki somewhere, because detailed list of features will be very useful.

**Comment by strixaluco** [comment]
Indeed. It appears that none of the markups can provide cell coloring in the tables on Github. 
What about using bold and italics? Something like this:

|  |  |  |  |
| --- | --- | --- | --- |
| [**XEP-0012: Last Activity**](http://www.xmpp.org/extensions/xep-0012.html) | [XEP-0016: Privacy Lists](http://www.xmpp.org/extensions/xep-0016.html) | [_XEP-0018: Invisible Presence_](http://www.xmpp.org/extensions/xep-0018.html) | [XEP-0022: Message Events](http://www.xmpp.org/extensions/xep-0022.html) |

**Comment by silverhook** [comment]
[XEP-0375: XMPP Compliance Suites 2016][375] might be a very good reference here. It specifies the 2016 compliance levels for XMPP clients and servers and thus what is the minimum requirement for a compliant server and client since 2016.

[375]: http://xmpp.org/extensions/xep-0375.html

**Comment by Neustradamus** [comment]
Any news on it?

A long time ago, I have create this issue: https://github.com/TelepathyIM/telepathy-gabble/issues/8

And there is now XEP-0412: XMPP Compliance Suites 2019:
- https://github.com/TelepathyIM/telepathy-gabble/issues/12

Do not forget: SCRAM:
- https://github.com/TelepathyIM/telepathy-gabble/issues/11
